src/helpers/apiCalls.js

```diff
@@ -1,4 +1,4 @@
-import { LOGIN_USER, SIGNUP_USER } from '../context/constants';
+import { FETCH_RECORDS, LOGIN_USER, SIGNUP_USER } from '../context/constants';
```

Comment:
is it best practice to write it in capitalls?

Reply:
It is convention for the type of the actions for useReducer, it comes from something `redux` started and useReducer is based on that.

Reply:
ok, danke